server/remote_cache/action_cache_server/action_cache_server.go

```diff
@@ -27,6 +28,8 @@ import (
 	rspb "github.com/buildbuddy-io/buildbuddy/proto/resource"
 )
 
+var checkClientActionResultDigests = flag.Bool("cache.check_client_action_result_digests", false, "If true, the server will check the bb-specific cached_action_result_digest field on ActionCache.getActionResult requests")
```

Comment:
nit: "...and omit them from the response proto."?

Reply:
fixed it up a bit